apps/cli/src/commands/create.ts

```diff
@@ -61,14 +62,26 @@ export default class CreateCommand extends Command {
         });
     }
 
+    private getRepoCodeLoadUrl(repository: string) {
+        return repository.replace("github.com", "codeload.github.com");
+    }
+
+    private getTarUrl(repository: string) {
+        if (repository === DEFAULT_TEMPLATE_REPOSITORY) {
+            return `${this.getRepoCodeLoadUrl(repository)}/tar.gz/refs/heads/${DEFAULT_TEMPLATES_BRANCH}`;
+        }
+        return `${this.getRepoCodeLoadUrl(repository)}/tar.gz/refs/heads/main`;
+    }
+
```

Comment:
This will be too error prone, since it will only work for github.

It looks like giget support a lot of other providers.

https://github.com/unjs/giget?tab=readme-ov-file#downloadtemplatesource-options